.travis.yml

```
@@ -1,12 +1,20 @@
 language: elixir
 elixir:
-  - 1.3.2
+  - 1.7.4
+  - 1.8.2
+  - 1.9.4
+
+cache:
+  directories:
+    - _build
+    - deps
+
 notifications:
-  recipients:
-    - travis@jeffweiss.org
+  email: true
+
 otp_release:
-  - 19.0
-env:
-  - MIX_ENV=test
+  - 20.0
+
 script:
   - "mix do deps.get, compile"
```

Podrías agregar la opción `--warning-as-errors` para que el CI truene si hay warnings
```suggestion
  - "mix do deps.get, compile --warning-as-errors"
```

---

Si, estoy de acuerdo. Aunque los únicos warnings que quiero eliminar son los que aparecen en Vanex cuando se compila y hace referencia al `mix.ex` de forcex.